x-pack/test_serverless/api_integration/services/alerting_api.ts

```diff
@@ -140,5 +140,16 @@ export function AlertingApiProvider({ getService }: FtrProviderContext) {
         });
       return body;
     },
+
+    async findRule(ruleId: string) {
+      if (!ruleId) {
+        throw new Error(`'ruleId' is undefined`);
+      }
+      const response = await supertest
+        .get(`/api/alerting/rule/${ruleId}`)
```

Comment:
@XavierM I am updating this to use the find api like this 
```
const response = await supertest
        .get('/api/alerts/_find')
        .set('kbn-xsrf','some-xsrf-token')
        .set('x-elastic-internal-origin', 'kibana');
```

but here's the error I get:

`'uri [https://localhost:5620/api/alerts/_find?per_page=20] with method [get] exists but is not available with the current configuration'`. Any idea how I could solve?

Reply:
you should use the `/api/alerting/rules/_find`.

Reply:
Ok it works now, so I can use the find api